app/routes.jsx

```diff
@@ -33,6 +34,8 @@ export default () => (
   <Switch>
     <Route exact path="/" component={HomePage} />
     <Route path="/about" component={About} />
+    <Route path="/resource_guides/:id" component={ResourceGuide} />
+    <Route path="/resource_guides" component={ResourceGuides} />
```

Comment:
Nit: Use dashes instead of underscores in URLs, since they're better for SEO purposes. Dashes are treated as word separators, while underscores cause the whole thing to be treated as a single word.